### PR TITLE
docs(website): initial helm guide article

### PR DESCRIPTION
**Pull request by jakepage91**
Closes # 

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

**Comment by jakepage91** [comment]
Dev to thumbnails (27)
Dev to thumbnails (26)

**Comment by jakepage91** [comment]
Dev to thumbnails (28)
Dev to thumbnails (29)

**Review by pmig — CHANGES_REQUESTED** [review]
Looks good, I would suggest also linking to our 5 shortcomings of helm article. Either in the dependency management warning section as "this is one of the shortcomings we explained on our other article about helm" or in the conclusion. Or maybe even both.

Maybe we can also add some more internal links. E.g. validating yaml files we can link to the complete kubectl guide :-)

**Comment by jakepage91** [comment]
> Looks good, I would suggest also linking to our 5 shortcomings of helm article. Either in the dependency management warning section as "this is one of the shortcomings we explained on our other article about helm" or in the conclusion. Or maybe even both.
> 
> Maybe we can also add some more internal links. E.g. validating yaml files we can link to the complete kubectl guide :-)

Done!

**Review by pmig — CHANGES_REQUESTED** [review]
Last small request:

I suggest alway`/blog/5-helm...` instead of `https://glasskube.dev/blog..` this ensures a easier top level domain switch and for local dev links don't break out ot a sudden :-)

File: website/blog/2024-10-03-helm-guide/index.mdx [hunk 1]
```diff
@@ -0,0 +1,597 @@
...
+-   **Secrets deserve extra protection:** Kubernetes secrets, despite their name, require additional safeguards. Employ like [helm-screts](https://github.com/jkroepke/helm-secrets) or [sealed-secrets](https://github.com/glasskube/packages/tree/main/packages/sealed-secrets) to encrypt and manage your sensitive data, adding an extra layer of security to your deployments.
+    
+
+## FAQ’s
+
+### Where are Helm charts stored?
+
+Helm charts can be stored in a few different places:
+
+**Remote Repositories:** These are online , mostly on GitHub, that host publicly available Helm charts. You can also find charts on GitLab, BitBucket, and other code hosting platforms too. For private Helm repository hosting, [Glasskube](https://glasskube.dev/) is a good option.
```
Comment (pmig) [inline website/blog/2024-10-03-helm-guide/index.mdx:457]: also here

Comment (jakepage91) [reply website/blog/2024-10-03-helm-guide/index.mdx]: Done!

**Review by thekubernaut — APPROVED** [review]

**Pull request merged**